src/main/resources/db-scripts/clickhouse/clickhouse.sql

```diff
@@ -31,13 +14,40 @@ CREATE TABLE sample_to_gene_panel
 ) ENGINE = MergeTree()
 ORDER BY (gene_panel_id, alteration_type, sample_unique_id);
 
-CREATE TABLE gene_panel_to_gene
+INSERT INTO sample_to_gene_panel_derived
+SELECT
+    concat(cs.cancer_study_identifier, '_', sample.stable_id) AS sample_unique_id,
+    genetic_alteration_type AS alteration_type,
+    -- If a mutation is found in a gene that is not in a gene panel we assume Whole Exome Sequencing WES
+    ifnull(gene_panel.stable_id, 'WES') AS gene_panel_id,
+    cs.cancer_study_identifier AS cancer_study_identifier
+FROM sample_profile sp
+         INNER JOIN genetic_profile gp ON sample_profile.genetic_profile_id = gp.genetic_profile_id
+         LEFT JOIN gene_panel ON sp.panel_id = gene_panel.internal_id
+         INNER JOIN sample ON sp.sample_id = sample.internal_id
+         INNER JOIN cancer_study cs ON gp.cancer_study_id = cs.cancer_study_id;
+
+CREATE TABLE gene_panel_to_gene_derived
 (
     gene_panel_id LowCardinality(String),
     gene String
 ) ENGINE = MergeTree()
 ORDER BY (gene_panel_id);
 
+INSERT INTO gene_panel_to_gene_derived
+SELECT
+    gp.stable_id AS gene_panel_id,
+    g.hugo_gene_symbol AS gene
+FROM gene_panel gp
+         INNER JOIN gene_panel_list gpl ON gp.internal_id = gpl.internal_id
+         INNER JOIN gene g ON g.entrez_gene_id = gpl.gene_id
+UNION ALL
+SELECT
+    'WES' AS gene_panel_id,
+    gene.hugo_gene_symbol AS gene
+FROM gene
+WHERE gene.entrez_gene_id > 0;
```

Comment:
some of our databases use negative entrez_gene_id values (mainly for microRNAmature/precursor combinations related to expression profiling). I'm not sure if that matters here. Maybe a note should be added about the possible future need to support negative values? Or maybe those are intentionally being excluded here.